migration/identity-migration/src/main/java/io/camunda/migration/identity/GroupMigrationHandler.java

```diff
@@ -45,13 +46,24 @@ protected void process(final List<Group> batch) {
 
   private MigrationStatusUpdateRequest processTask(final Group group) {
     try {
-      // TODO: Revisit this part with https://github.com/camunda/camunda/issues/26973
-      //      groupServices.createGroup(group.name()).join();
+      final var groupDTO = new GroupDTO(normalizeGroupID(group.name()), group.name(), "");
```

Comment:
🔧  wdyt of this way for handling potential empty outcome of normalization?
```suggestion
      final var groupDTO = new GroupDTO(Optional.of(normalizeGroupID(group.name())).filter(s -> !s.isEmpty()).orElse(group.id()), group.name(), "");
```

Reply:
The result of the normalization  can't be empty, but I added a check to fallback to the id in case the group name is empty